app/assets/javascripts/kassi.js

```diff
@@ -1065,7 +1068,7 @@ function set_textarea_maxlength() {
 
       }
     })
-    .live('keydown', function(event) {
+    .on('keydown', function(event) {
```

Comment:
Wasn't this live stuff something about event handlers working even when elements inserted dynamically (ajax) later? Is this an issue here?